src/pages/documentation/release-notes/2021.md

```diff
@@ -77,7 +77,7 @@ For more details, see the documentation and release notes in the [aepsdk-react-n
 
 ### iOS AEPMobileServices 3.0.3
 
- * Fixes crash when using an immutable dictionary for handling attribution data.
+* Fixes crash when using an immutable dictionary for handling attribution data.
```

Comment:
I think it doesn't matter which one you use, so long as you're consistent within the file itself. So I think in these cases, it's just going with the one that's appearing most often within the file and getting rid of ones where there's both the `-` and the `*`.

For what it's worth, I don't really care which one people use - I personally prefer `-`, but it doesn't really matter which one we use so long as we're consistent between them all.

Reply:
For some reason I like `*` more, but either way I think we should aim for consistency. 
I noticed that the markdownlint config uses list style "consistent", which seems to take the consistency in the nearby modified lines rather than the entire file. This way we may still end up with a mix of both and doc writers will run into lint errors unexpectedly.